rollup/core/objects/RollupLoggerPluginParameter__mdt/fields/Value__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Value__c</fullName>
+    <description>Generic text value - different records can hold different param values here</description>
+    <externalId>false</externalId>
+    <fieldManageability>SubscriberControlled</fieldManageability>
+    <inlineHelpText>Generic text value - different records can hold different param values here</inlineHelpText>
+    <label>Value</label>
+    <length>255</length>
```

Comment:
Text(255) seeeeems like enough space for your purposes, right? I used long text area for some of the Nebula stuff, but it doesn't seem necessary for your purposes.

Reply:
I feel pretty confident in that limit 😁:

```js
'SupercalifragilisticexpalidociousSuperManagerFactoryBuilderFacade'.length // prints 65
```